### PR TITLE
fix(docs): correct typo mistakes

### PR DESCRIPTION
**Pull request by steffinstanly**
# Issue #150
<OVERALL SUMMARY OF PULL REQUEST>

### Changes

Fixed typo mistakes

### Flags
- <DESCRIBE ISSUES OR HOLDS FOR REVIEWERS>

### Related Issues
- Issue #<NUMBER HERE>
- Pull Request #<NUMBER HERE>

### CODE REVIEW

File: docs/advanced-latedelivery.md [hunk 1]
```diff
@@ -212,7 +212,7 @@ asset MiniLateDeliveryContract extends AccordContract {
 }
 ```
 
-The next error is in the logic, since it still uses the old `MiniLateDeliveryClause` type which does not exist anymore.
+The next error is in the logic since it still uses the old `MiniLateDeliveryClause` type which does not exist anymore.
```
Comment (irmerk) [inline docs/advanced-latedelivery.md]: This can be reverted.

File: docs/advanced-latedelivery.md [hunk 2]
```diff
@@ -23,17 +23,17 @@ Then head to the `Template` tab in the `Contract Text` section which shows the n
...
+When the text in the `Template` tab is in sync with the text in the `Test Contract` tab, this results in a valid contract instance in the `Contract Data` tab. The contract data is represented using the JSON format and contains the value of the variables declared in the contract template. For instance, the value for the `buyer` variable is `Betty Buyer`, highlighted in red:
 
 ![Advanced-Late-3](assets/advanced/late3.png)
 
-Changes to the variables in the `Test Contract` are reflected in the Contract Data in real time, and vice versa. For instance, change `Betty Buyer` to a different name in the contract text to see the `partyId` change in the contract data.
+Changes to the variables in the `Test Contract` are reflected in the Contract Data in real-time, and vice versa. For instance, change `Betty Buyer` to a different name in the contract text to see the `partyId` change in the contract data.
```
Comment (irmerk) [inline docs/advanced-latedelivery.md]: This can be reverted.

Comment (jeromesimeon) [reply docs/advanced-latedelivery.md]: Why is it `real-time` in one place and `realtime` in another in this PR?

Comment (steffinstanly) [reply docs/advanced-latedelivery.md]: sorry, my mistake I fixed it now.

**Pull request merged**